README.md

```diff
@@ -1,182 +1,176 @@
-
-If a module is modified, `import` will
-realize this and reload the script if further imports are executed or 
-re-executed; otherwise additional imports will not cause the script to be 
-reloaded for efficiency. As the script is loaded in its own environment 
-(maintained by `import`) dependencies are kept (except those exposed through
-attachment), as the following small example shows.
-
-**Contents of "some_module.R":**
-```R
-## Possible
-library(ggplot2)
-
-## But would be better:
-#import::here(qplot, .from = ggplot2)
-
-## Note this operator overload is not something you want to `source`!
-`+` <- function(e1, e2)
-  paste(e1, e2)
+When importing from a module (.R file), the directory where `import` looks for
+the module script can be speecified with the with `.directory` paremeter. 
+The default is `.` (the current working directory).
 
-## Some function relying on the above overload:
-a <- function(s1, s2)
-  s1 + rep(s2, 3)
+### Choosing where the imported functions are placed
 
-## Another value.
-b <- head(iris, 10)
+One can also specify which names to use in the search path and use several to 
+group imports:
 
-## A value created using a function exposed by attachment 
-p <- qplot(Sepal.Length, Sepal.Width, data = iris, color = Species)
+```R
+import::from(magrittr, "%>%", "%$%", .into = "operators") 
+import::from(dplyr, arrange, .into = "datatools")
+```
 
-## A function relying on a function exposed through attachment:
-plot_it <- function()
-  qplot(Sepal.Length, Sepal.Width, data = iris, color = Species)
+If using pied-à-terres actively, one might prefer the alternative syntax 
+(which does the same but reverses the argument order):
 
+```R
+import::into("operators", "%>%", "%$%", .from = magrittr)
+import::into("datatools", arrange, .from = dplyr)
 ```
+If it is desired to place imported objects in the current environment, 
+use `import::here()`:
 
-**Usage:**
-```R
-import::from(some_module.R, a, b, p, plot_it)
+### More advanced usage
 
-## Works:
-a("cool", "import")
+The `import` package is designed to be simple to use for basic cases, so it uses
+symbolic evaluation to allow the names of packages, modules and functions to be
+entered without quotes (except for operators, such as `"%>%"` which must be quoted).
+However, this means that it calling a variable containing the name of a module, or a 
+vector of functions to import, will not work. For this use case, you can use the
+`.character_only` parameter:
 
-## The `+` is not affecting anything here, so this won't work:
-# "cool" + "import"
+```R
+module_name <- "../utils/my_module.R"
 
-# Works:
-b
-p
+# Will not work (import will look for a package called "module_name")
+import::from(module_name, foo, bar)
 
-## Does not work, as ggplot2 is no longer attached
-## (would work with the import statement!):
-#plot_it()
+# This will correctly import the foo() and bar() functions from "../utils/my_module.R"
+import::from(module_name, foo, bar)
```

Comment:
is this not the same as above?

Reply:
Added `.character_only=TRUE` to the second instance, which was supposed to be the difference.